types/api.go

```diff
@@ -568,16 +568,30 @@ type ApiValidatorEth1Response struct {
 }
 
 type ApiValidatorIncomeHistoryResponse struct {
-	Income struct {
-		AttestationSourceReward uint64 `json:"attestation_source_reward"`
-		AttestationTargetReward uint64 `json:"attestation_target_reward"`
-		AttestationHeadReward   uint64 `json:"attestation_head_reward"`
-	} `json:"income"`
-	Epoch          uint64    `json:"epoch"`
-	ValidatorIndex uint64    `json:"validatorindex"`
-	Week           uint64    `json:"week"`
-	WeekStart      time.Time `json:"week_start"`
-	WeekEnd        time.Time `json:"week_end"`
+	Income         *ApiValidatorIncomeHistory `json:"income"`
+	Epoch          uint64                     `json:"epoch"`
+	ValidatorIndex uint64                     `json:"validatorindex"`
+	Week           uint64                     `json:"week"`
+	WeekStart      time.Time                  `json:"week_start"`
+	WeekEnd        time.Time                  `json:"week_end"`
+}
+
+type ApiValidatorIncomeHistory struct {
```

Comment:
I am not sure whether I like duplicating the info here (compared with `itypes "github.com/gobitfly/eth-rewards/types"` where it came before) or not.
Changes in `itypes "github.com/gobitfly/eth-rewards/types"` now will not break the API but it feels redundant.
Could you please share your thoughts?